website/client/src/components/groups/questDetailModal.vue

```diff
@@ -0,0 +1,450 @@
+<template>
+  <b-modal
+    id="quest-detail-modal"
+    title="Empty"
+    size="md"
+    :hide-footer="true"
+    :hide-header="true"
+    :modal-class="dialogClass"
+  >
+    <div class="dialog-close">
+      <close-icon @click="close()" />
+    </div>
+    <h2 class="text-center textCondensed">
+      {{ selectMode ? $t('selectQuest') : $t('questDetailsTitle') }}
+    </h2>
+    <div class="quest-panel" v-if="selectMode">
+      <div class="quest-panel-header">
+        <h3>
+          {{ $t('yourQuests') }}
+        </h3>
+        <div class="sort-by">
+          <span class="dropdown-label">{{ $t('sort') }}</span>
+          <select-translated-array
+            :right="true"
+            :items="['quantity', 'AZ']"
+            :value="sortBy"
+            class="inline"
+            :inline-dropdown="false"
+            @select="sortBy = $event"
+          />
+        </div>
+      </div>
+
+      <div class="quest-items">
+        <!-- eslint-disable vue/no-use-v-if-with-v-for -->
```

Comment:
You don't have a `v-if` in here anyway, no need to disable this lint

Reply:
since this whole file was "re-added" and not moved .... (wonder why)

+ refactoring this was probably from previous code I assume - will be removed